### PR TITLE
[onert-micro] support weight quantized (int8) FullyConnected kernel

### PR DESCRIPTION
**Pull request by Torrero**
This PR adds support of the weight quantized FullyConnected kernel

- FullyConnected kernel (input:FLOAT32 + weights:INT8)

ONE-DCO-1.0-Signed-off-by:  Evgenii Maltsev e.maltsev@samsung.com

### CODE REVIEW

File: onert-micro/onert-micro/src/import/kernels/FullyConnected.cpp [hunk 1]
```diff
@@ -60,13 +60,56 @@ onert_micro::import::configure_kernel_CircleFullyConnected(const OMConfigureArgs
...
+  if (weight->type() == circle::TensorType_UINT8)
+  {
+
+    status = utils::checkCondition(input->type() == circle::TensorType_UINT8 and
+                                   output->type() == circle::TensorType_UINT8 and
+                                   (!bias or bias->type() == circle::TensorType_INT32));
+    if (status != Ok)
+      return status;
+  }
+  else if (weight->type() == circle::TensorType_INT8)
```
Comment (stamalakhov) [inline onert-micro/onert-micro/src/import/kernels/FullyConnected.cpp:84]: Is INT16 no longer supported ?

**Review by stamalakhov — APPROVED** [review]
LGTM! Thank you!

File: onert-micro/onert-micro/src/import/kernels/FullyConnected.cpp (hunk 1 above)
Comment (Torrero) [reply onert-micro/onert-micro/src/import/kernels/FullyConnected.cpp:84]: Added this check for INT16, thank you

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**